setup.cfg

```diff
@@ -113,3 +113,13 @@ python-tag = py38
 omit =
     src/sentry/migrations/*
 source = .
+
+[coverage:report]
+exclude_lines =
+    # Once we upgrade to coverage >= 7.2, we can switch to using `exclude_also` and
+    # remove `"pragma: no cover"`. See https://coverage.readthedocs.io/en/latest/config.html#report-exclude-also.
+    "pragma: no cover",
+
+    # This may at some point get added to the defaults, in which case we can remove it
+    # from our config. See https://github.com/nedbat/coveragepy/issues/831.
+    "if TYPE_CHECKING:",
```

Comment:
we should use https://github.com/asottile/covdefaults

Reply:
Oh, nice!

So, what would I do, install it, then `pip freeze` into `requirements-dev.text`? What's the deal with the frozen version?

(I get that I have to add it into the config, also, but that part is straightforward. I'm also fine just closing this if it's quicker for you to do it.)

Reply:
https://develop.sentry.dev/python-dependencies/